app/controllers/attachments_controller.rb

```diff
@@ -3,9 +3,15 @@ class AttachmentsController < BaseAttachmentsController
 
   def show
     unless user_signed_in?
-      asset_host = URI.parse(Plek.new.public_asset_host).host
+      asset_url = if request.host.start_with?('draft-')
```

Comment:
Could we extract this block of code into a common class or method as it's the same as the code in the `AssetManagerRedirectController` above and would be good to have it testable in one place?

Reply:
Yeah true, probably not worth it.